README.md

```diff
@@ -49,18 +49,15 @@ If you are on NixOS, use [shell.nix](shell.nix) to pull the dependencies and set
 
 ### Commands
 
-Check `<latest release tag>` on https://github.com/input-output-hk/jormungandr/releases/latest
+Get the latest "master" branch of the repository and install
```

Comment:
It is not always advisable for end users to run from the master branch, as there could be incompatible changes from previous versions that are unexpected for non-dev (users who are not sure what to make of changelog)

Reply:
When building from source it is better to checkout a version number, indeed. Otherwise using the master branch for development is better. I prefer to still advise users to use latest master for now (as @rdlrt suggested). It is safer, especially on the long run.

Reply:
(note that I write "mechanically")

* Cardano Shelly docu, installs from `master` => you need to be consistent with this (that's why I change it)
* If you want to install `latest-stable` => remove the need for manual lookup (see #1534)
  * Ensure that Carano Shelly docs get update to use `latest-stable`, too
* Developers know anyways what to do